content/blog/wingingit/wingingit.11tydata.yaml

```diff
@@ -0,0 +1,6 @@
+venue: Winging It
+url: https://www.youtube.com/@wingingitoddbird
```

Comment:
Since venue is a location , it controls the meta prefix used on the byline ... so right now it say:

**by Miriam, Ed, & Stacy at Winging It on July 9, 2023**

But I think it would sound better to say 

**by Miriam, Ed, & Stacy on Winging It posted July 9, 2023**

I could change the venue name and it would say 

**by Miriam, Ed, & Stacy at The Winging It Channel on July 9, 2023**

Anyone have any strong feelings about this?

Reply:
@stacyk I have a slight preference for 
by Miriam, Ed, & Stacy at The Winging It Channel on July 9, 2023

Reply:
@jgerigmeyer you may want to check my work since I added the length to the post list card and the post detail. I left it a string incase we ever have longer episodes and wanted more flexibility in how we display that. We may want to remove the space between the time and the label?

![length-post-detail](https://github.com/oddbird/oddleventy/assets/1581694/9c812a1b-4ab0-4308-93f8-40798bdf68f1)
![length-post-list](https://github.com/oddbird/oddleventy/assets/1581694/00fe3c76-c1c3-485a-875c-1cbf40e6c1c9)

Reply:
@stacyk Looks good! I pushed minor cleanup; up to you about the spacing?